entity-types/ext-temporal/definition.yml

```diff
@@ -0,0 +1,28 @@
+---
+domain: EXT
+type: TEMPORAL
+synthesis:
+  identifier: temporal_identifier
```

Comment:
The validator is complaining because you can't define a rule here (by providing the `identifier` and `name` while there is a list of `rules` defined.

Reply:
Hi, @javimb ,
Removed the identifier and name. Could you please review it?